diff -r -u a/gitolite/src/lib/Gitolite/Rc.pm b/gitolite/src/lib/Gitolite/Rc.pm
--- a/gitolite/src/lib/Gitolite/Rc.pm	2024-07-06 15:35:03.949676184 -0700
+++ b/gitolite/src/lib/Gitolite/Rc.pm	2024-07-07 13:11:00.703642175 -0700
@@ -40,7 +40,7 @@
 # these keys could be overridden by the rc file later
 $rc{GL_REPO_BASE}  = "$ENV{HOME}/repositories";
 $rc{GL_ADMIN_BASE} = "$ENV{HOME}/.gitolite";
-$rc{LOG_TEMPLATE}  = "$ENV{HOME}/.gitolite/logs/gitolite-%y-%m.log";
+$rc{LOG_TEMPLATE}  = "/var/tmp/gitolite.log";
 
 # variables that should probably never be changed but someone will want to, I'll bet...
 # ----------------------------------------------------------------------
@@ -105,7 +105,7 @@
     # it down to all its children so you can track each access, across all the
     # various commands it spawns and actions it generates.
 
-    $rc{GL_LOGFILE} = $ENV{GL_LOGFILE} ||= gen_lfn( $rc{LOG_TEMPLATE} );
+    $rc{GL_LOGFILE} = $ENV{GL_LOGFILE} ||= $rc{LOG_TEMPLATE};
 }
 
 # these two are meant to help externally written commands (see
